### PR TITLE
DOCSP-41008: v2.27 release

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-41008
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/csharp/DOCSP-41008-2.27-release/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -30,6 +31,31 @@ Learn what's new in:
...
+
+- Adds support for the ``$sample`` aggregation operator.
+
+- Implements the ``Equals()`` method for serializers in LINQ3.
+
+- Includes tests confirming that read and write concerns are not
+  applied to Atlas Search Index Helper commands.
+
+- Disallows the use of the comma character in ``authMechanismProperties``
+  connection string values.
```
Comment (rustagir) [inline source/whats-new.txt:49]: S: Is there a separate ticket to add this to the OIDC documentation? It should be more clear here also that this is an OIDC-specific change. See Java PR for details: https://github.com/mongodb/docs-java/pull/553

**Review by rustagir — COMMENTED** [review]
Left a comment to address, can take another look after you respond!

File: source/whats-new.txt (hunk 1 above)
Comment (norareidy) [reply source/whats-new.txt]: I made a ticket to add that info: https://jira.mongodb.org/browse/DOCSP-41035

**Review by rustagir — APPROVED** [review]
lgtm!

File: source/whats-new.txt [hunk 2]
```diff
@@ -30,6 +31,32 @@ Learn what's new in:
...
+What's New in 2.27
+------------------
+
+The 2.27 driver release includes the following new features:
+
+- Adds support for the ``$sample`` aggregation operator.
+
+- Implements the ``Equals()`` method for serializers in LINQ3.
+
+- Includes tests confirming that read and write concerns are not
```
Comment (BorisDog) [inline source/whats-new.txt:45]: Not just tests.
"Avoid appending Write/Read Concern in Atlas Search Index Helper commands"

**Review by BorisDog — CHANGES_REQUESTED** [review]

File: source/whats-new.txt (hunk 2 above)
Comment (norareidy) [reply source/whats-new.txt]: Reworded - let me know what you think!

**Review by BorisDog — APPROVED** [review]
LGTM

**Pull request merged**